src/main/resources/templates/fragments/entry-table.html

```diff
@@ -4,6 +4,10 @@
 <body>
 <div th:fragment="entry-table(content)">
     <table class="entry">
+        <colgroup>
+            <col class="field-column" />
```

Comment:
fyi our templating language (thymeleaf) is xml based so these trailing `/` characters are required. they won't appear in the html5 output.